src/components/sections/items/GenericClickableSectionItem.tsx

```diff
@@ -2,15 +2,20 @@ import React, {PropsWithChildren, forwardRef} from 'react';
 import {AccessibilityProps, View} from 'react-native';
 import {useSectionItem} from '../use-section-item';
 import {SectionItemProps} from '../types';
-import {PressableOpacity} from '@atb/components/pressable-opacity';
+import {
+  PressableOpacity,
+  PressableOpacityProps,
+} from '@atb/components/pressable-opacity';
 
 type Props = PropsWithChildren<
   SectionItemProps<
     {
       onPress?(): void;
-    } & AccessibilityProps
+    } & AccessibilityProps &
+      PressableOpacityProps
```

Comment:
Maybe just add `disabled` instead of the full `PressableOpacityProps`? We want to expose parameters on a need-to-have basis.

In addition, could the opacity when disabled be handled inside `GenericClickableSectionItem`? So parent components that specify `disabled={true}` doesn't need to do the styling themselves.